format/spec.md

```diff
@@ -241,7 +241,13 @@ Struct evolution requires the following rules for default values:
 
 #### Column Projection
 
-Columns in Iceberg data files are selected by field id. The table schema's column names and order may change after a data file is written, and projection must be done using field ids. If a field id is missing from a data file, its value for each row should be `null`.
+Columns in Iceberg data files are selected by field id. The table schema's column names and order may change after a data file is written, and projection must be done using field ids.
+
+Values for Field ids which are not present in a data file must be resolved according the following rules:
+
+* Return the value from partition metadata if an [Identity Transform](#partition-transforms) exists for the field. 
```

Comment:
> Return the value from partition metadata if the data file's partition metadata contains the field as a result of the `identity` transform.

I want to be clear that this is only required if the data file's partition metadata contains the field, not just if there is a partition spec that uses an identity transform for the field.

Reply:
I think I addressed this but not sure if I fully understood the comment.